zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/ziplineFunctions.kt

```diff
@@ -48,6 +50,7 @@ internal abstract class SuspendingZiplineFunction<T : ZiplineService>(
 ) : ZiplineFunction<T> {
   val argsListSerializer = ArgsListSerializer(argSerializers)
 
+  override val isSuspending = true
   override val isClose = false
```

Comment:
```suggestion
  override val isSuspending get() = true
  override val isClose get() = false
```

Reply:
I’d considered this. It generates better bytecode at a cost of _sliiiiightly_ larger sourcecode. Probably a good tradeoff considering that the sourcecode has fewer moving parts.